### PR TITLE
utils 0.1.0-dev

### PR DESCRIPTION
**Pull request by mgharbour**
Created via `alr publish` with `alr 2.0.1`

### CODE REVIEW

File: index/ut/utils/utils-0.1.0-dev.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+authors = ["Michael Gonzalez"]
+maintainers = ["Michael Gonzalez <mgh@unican.es>"]
+maintainers-logins = ["mgharbour"]
+licenses = "GPL-3.0-or-later"
+website = "mast.unican.es"
+tags = ["mast", "schedulability", "analysis", "real-time"]
+
+[origin]
+commit = "92c0c10b3aa8a85ee63fba7ffea7713d9cbf6841"
+url = "git+https://github.com/mgharbour/mast.git"
```
Comment (Fabien-Chouteau) [inline index/ut/utils/utils-0.1.0-dev.toml:14]: Looks like the repo is not public. So Alire won't be able to download the sources.

File: index/ut/utils/utils-0.1.0-dev.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name = "utils"
+description = "mast utilities"
+version = "0.1.0-dev"
```
Comment (Fabien-Chouteau) [inline index/ut/utils/utils-0.1.0-dev.toml:3]: For an Alire release you probably don't want to keep the `-dev` suffix.

**Review by Fabien-Chouteau — CHANGES_REQUESTED** [review]
Hi @mgharbour , welcome to Alire.

There are a couple of things to fix before we can merge this request.

**Comment by mosteo** [comment]
This one is not needed anymore per talks with @mgharbour

**Pull request closed without merging**